play-services-core/src/main/res/drawable-anydpi-v21/microg_light_color_24.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the source and license of this icon? Looks like it could be an original Google file (even though it's not very complex), so please verify it is not or at least properly licensed. Thanks.

Reply:
You're right. I just accepted the pull request over at my fork without further checking. I'll create a new drawable from the official microG icon.

Reply:
Added new drawable based on artwork/ic_microg_settings.svg, should be fine now.